3-4/GalilSup/Db/Makefile

```diff
@@ -11,6 +11,8 @@ include $(TOP)/configure/CONFIG
 # Create and install (or just install) into <top>/db
 # databases, templates, substitutions like this
 
+DB += $(patsubst ../%, %, $(wildcard ../*.template))
```

Comment:
Do you really need this?  I know its standard, but feel it doesn't add value.  It means the driver must be recompiled if a template is changed.  I usually load directly from Db directory.